client/app/telecom/telephony/alias/portability/order/telecom-telephony-alias-portability-order.controller.js

```diff
@@ -1,242 +1,255 @@
-    } else {
-      params = _.assign(params, _.pick(self.order, 'siret'));
-    }
-    params.firstName = params.firstName || '';
-
-    if (self.isSDA && self.order.sdatype === 'select' && self.order.numbersList.length) {
-      params.listNumbers = _.map(self.order.numbersList, self.normalizeNumber).join(',');
-    }
-
-    if (self.isSpecialNumber) {
-      params.specialNumberCategory = self.order.specialNumberCategory.replace('fr_', '');
-    }
-
-    if (params.country === 'france') {
-      params.fiabilisation = self.order.sdatype === 'all';
-    }
-
-    params.callNumber = self.normalizeNumber(params.callNumber);
-    params.contactNumber = self.normalizeNumber(params.contactNumber);
-    params.desireDate = moment(params.desireDate).format('Y-MM-DD');
-
-    return params;
-  };
-
-  self.fetchPriceAndContracts = function () {
-    self.step = 'summary';
-    return OvhApiOrder.Telephony().v6()
-      .getPortability(self.getOrderParams()).$promise
-      .then((result) => {
-        self.details = result.details;
-        self.contracts = result.contracts;
-        self.prices = result.prices;
-      })
-      .catch((err) => {
-        self.step = 'number';
-        TucToast.error(_.get(err, 'data.message'));
-        return $q.reject(err);
-      });
-  };
-
-  self.submitOrder = function () {
-    self.order.isOrdering = true;
-    return OvhApiOrder.Telephony().v6().orderPortability({
-      billingAccount: self.order.billingAccount,
-    }, _.omit(self.getOrderParams(), 'billingAccount')).$promise.then((result) => {
-      self.order.success = true;
-      self.order.url = result.url;
-      return OvhApiMe.Order().v6().get({
-        orderId: result.orderId,
-      }).$promise.then(
-        () => OvhApiMe.Order().v6().payRegisteredPaymentMean({
-          orderId: result.orderId,
-        }, {
-          paymentMean: 'ovhAccount',
-        }).$promise.then(() => {
-          self.order.autoPay = true;
-        }, () => {
-          // if it fails no need to reject because portablity order is a success
-          // and validation can always be done by clicking
-          self.order.autoPay = false;
-        }),
-        () => {
-        // in this case it means that nic bill and connected are not the same
-        // so display a message telling that order must be validated by clicking
-        // no need to reject because portablity order is a success
-        // and validation can always be done by clicking
-          self.order.autoPay = false;
-        },
-      );
-    }).catch((err) => {
-      TucToast.error([$translate.instant('telephony_alias_portability_order_error'), _.get(err, 'data.message')].join(' '));
-      return $q.reject(err);
-    }).finally(() => {
-      self.order.isOrdering = false;
+      self.onSDATypeChange = function () {
+        self.order.socialReason = self.isSDA ? 'corporation' : 'individual';
+      };
+
+      self.openDesireDatePicker = function (ev) {
```

Comment:
I know that `ev` stands for event, but it should be not abbreviated

Reply:
previous code